app/routes/docs.team/route.tsx

```diff
@@ -53,51 +42,34 @@ function TeamCard({
   affiliation,
   photo,
   href,
-  tags,
-  selectedTags,
 }: {
   children: ReactNode
   name: string
   affiliation: string
   photo: string
   href: string
-  tags: string[]
-  selectedTags: string[]
 }) {
-  const tagSet = new Set(tags)
   return (
-    <>
-      {(selectedTags.length == 0 ||
-        selectedTags.every((tag) => tagSet.has(tag))) && (
-        <Card className="grid-col-fill">
-          <CardMedia imageClass="bg-white margin-bottom-neg-2">
-            <img
-              src={photo}
-              width="50"
-              height="50"
-              loading="lazy"
-              alt="headshot of GCN team member"
-            />
-          </CardMedia>
-          <CardHeader>
-            <h4 style={{ marginBottom: '0px' }}>
-              <a className="usa-link" href={href}>
-                <>{name}</>
-              </a>
-            </h4>
-            <p style={{ marginTop: '0px', fontSize: '12px' }}>{affiliation}</p>
-          </CardHeader>
-          <CardBody style={{ fontSize: '16px' }}>{children}</CardBody>
-          <CardFooter style={{ marginBottom: '0px' }}>
-            {tags?.map((tag) => (
-              <Tag key={tag} className="font-body-4xs">
-                {tag}
-              </Tag>
-            ))}
-          </CardFooter>
-        </Card>
-      )}
-    </>
+    <Card className="grid-col-fill">
+      <CardMedia imageClass="bg-white margin-bottom-neg-2">
+        <img
+          src={photo}
+          width="50"
+          height="50"
+          loading="lazy"
+          alt="headshot of GCN team member"
```

Comment:
A "headshot" has the unfortunate second meaning of a gunshot to the head.

```suggestion
          alt="Photograph of GCN team member"
```